header.php

```diff
@@ -30,6 +30,9 @@ function gtag(){dataLayer.push(arguments);}
 	<!-- Load Google Fonts -->
 	<link href="https://fonts.googleapis.com/css?family=Raleway:400,700" rel="stylesheet"> 
 
+	<!-- jQuery -->
```

Comment:
It was my understanding that jquery was already included on all pages? Is there a reason you include the slim library as well?

Reply:
No good reason, I'm not sure why I did that either. Unfortunately I didn't see this until after I committed it - I will fix this up today.